server/user.go

```diff
@@ -116,7 +116,7 @@ func (p *Plugin) StoreUserInfo(info *UserInfo) error {
 func (p *Plugin) GetUserInfo(userID string) (*UserInfo, error) {
 	infoBytes, appErr := p.API.KVGet(tokenKey + userID)
 	if appErr != nil || infoBytes == nil {
-		return nil, errors.New("must connect user account to Microsoft first")
+		return nil, errors.New("your Mattermost account is not connected to any Microsoft Teams account")
```

Comment:
This looks off not being capitalized here. I think CI enforces errors to have a lowercase letter at the beginning of the error. Can we either make this function return an additional string value to display to the user? Either that or disable the link rule on this line

Reply:
@mickmister disabled the golint for this line and capitalized the first letter.